source/operations/external-iam/configure-ad-ldap-external-identity-management.rst

```diff
@@ -150,99 +195,60 @@ An AD/LDAP user with no assigned policy *and* with membership in groups with no
             export MINIO_IDENTITY_LDAP_GROUP_SEARCH_FILTER="(&(objectClass=group)(member=%d))"
             export MINIO_IDENTITY_LDAP_GROUP_SEARCH_BASE_DN="ou=MinIO Users,dc=example,dc=net"
 
-         For complete documentation on these variables, see
-         :ref:`minio-server-envvar-external-identity-management-ad-ldap`
-
-      .. tab-item:: Configuration Settings
+            export MINIO_IDENTITY_LDAP_STS_EXPIRY="1h"
+            export MINIO_IDENTITY_LDAP_USERNAME_FORMAT="uid=%s,cn=miniousers,dc=myldapserver,dc=net,userPrincipalName=%s,cn=miniousers,dc=myldapserver,dc=net"
+	    export MINIO_IDENTITY_LDAP_TLS_SKIP_VERIFY="off"
+	    export MINIO_IDENTITY_LDAP_SERVER_INSECURE="off"
+	    export MINIO_IDENTITY_LDAP_SERVER_STARTTLS="off"
+	    export MINIO_IDENTITY_LDAP_COMMENT="LDAP test server"
 
-         MinIO supports specifying the AD/LDAP provider
-         settings using :mc-conf:`configuration settings <identity_ldap>`. The 
-         :mc:`minio server` process applies the specified settings on its next
-         startup. For distributed deployments, the :mc:`mc admin config`
-         command applies the configuration to all nodes in the deployment.
+         For complete documentation on these variables, see :ref:`minio-server-envvar-external-identity-management-ad-ldap`
 
-         The following example code sets *all* configuration settings related to
-         configuring an AD/LDAP provider for external
-         identity management. The minimum *required* setting are:
-         
-         - :mc-conf:`identity_ldap server_addr <identity_ldap.server_addr>`
+      .. tab-item:: MinIO Console
 
-         - :mc-conf:`identity_ldap lookup_bind_dn <identity_ldap.lookup_bind_dn>`
+         MinIO supports specifying the AD/LDAP provider settings using the :ref:`MinIO Console <minio-console>`.
```

Comment:
Does configuring LDAP from Console do it for the entire deployment? (Like the `mc` methods do.)

Reply:
I believe so - it should be the rough equivalent of using `mc` from an API perspective.